python/twitter_bot/readme.md

```diff
@@ -0,0 +1,68 @@
+# Pangolin Twitter bot
```

Comment:
should be named README.md like the rest of the readme files?

Reply:
Yes